sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -499,6 +502,24 @@ def test_importance_getter(estimator, importance_getter):
     assert selector.transform(data).shape[1] == 1
 
 
+# TODO(1.3): remove
+@pytest.mark.parametrize("PLSEstimator", [CCA, PLSCanonical, PLSRegression])
+def test_select_from_model_pls(PLSEstimator):
+    """Check the behaviour of SelectFromModel with PLS estimators.
+
+    Non-regression test for:
+    https://github.com/scikit-learn/scikit-learn/issues/12410
+    """
+    X, y = make_friedman1(n_samples=50, n_features=10, random_state=0)
+    estimator = PLSEstimator(n_components=1)
+    model = make_pipeline(SelectFromModel(estimator), estimator)
+    # We must not warn by accessing the `coef_` attribute of PLSEstimator
+    with warnings.catch_warnings():
+        warnings.simplefilter("error", FutureWarning)
+        model.fit(X, y)
+    assert model.score(X, y) > 0.5
```

Comment:
I'm not sure this test is necessary. It's only purpose is to check that SelectFromModel does not raise a FutureWarning, but if it did, our CI would catch it anyway. Do you agree to remove it ?

Reply:
I agree with the warning checking. However we can keep the fit and scoring because it was raising an error before.

Reply:
makes sense